src/arm/BONE-UART1-00A0.dts

```diff
@@ -0,0 +1,36 @@
+/*
```

Comment:
These filenames don't match the specification in the wiki: https://elinux.org/Beagleboard:BeagleBone_cape_interface_spec#.2Fdev.2Fbone.2Fuart.2F1

Reply:
I don't want to touch those as they are not interfering with these, people might want to use those. I was thinking of updating the links after the merge, else they will show 404 not found. Also, I am working on the tutorial/usage of these overlays :)